### ISSUE TITLE
UX: Improve functionality of layer select/de-select buttons

### ISSUE THREAD

**Issue by kidwellj** [mapcomm/common-ground_map#27]
Buttons should change their display to indicate somehow whether a layer is on or off. Can we add opacity when layer is turned off? We also need to alter colours to match colorbrewer selections for points, and add a title indicating to users that these buttons function as a "Legend" for the map.

**Comment by danhillcode** [comment]
so when selected the button will be bright and then when not selected it will be shaded is that correct?

**Comment by kidwellj** [comment]
Yeah, something like that. Here's an example of what I'm imagining:
<img width="319" alt="lh_sidebar_example" src="https://user-images.githubusercontent.com/790736/30249669-995fa6c6-9638-11e7-86c3-9230191178a0.png">

**Comment by danhillcode** [comment]
I have changed the opacity so that when a layer is on it has opacity on as only one layer can be on at a time anyway. Added some more css around the css to make it more styled I like it with the border.

**Comment by danhillcode** [comment]
might be an error here just testing but hard to tell without points on map will check again.

**Comment by kidwellj** [comment]
That's (nearly) good for now - but need to reverse styling so that other buttons are off and active button is 100% black. Sorry - wasn't thinking about the fact that we can only select one set at a time originally.

**Comment by danhillcode** [comment]
No worries.
